request/auth.php

```diff
@@ -14,7 +14,7 @@
                     case 'login':
                         if (!empty($_POST['email']) && !empty($_POST['password'])) {
                             if (FirstAide\Utils::isValidEmail($_POST['email'])) {
-                                $Auth = FirstAide\Authentication::withEmailPassword(
+                                $Auth = FirstAide\Authentication::createInstanceWithEmailPassword(
```

Comment:
Lets move this logic to a separate method that validates and returns a boolean and then call the method in the if condition. 
```
if (isset($_POST)) {
        if (isset($_POST['csrf_token']) && $_POST['csrf_token'] == $csrf_token) {
  }
}
```